pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -58,6 +59,24 @@ def test_cov(self, float_frame, float_string_frame):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("test_ddof", [0, 1, 2, 3])
+    def test_cov_ddof(self, test_ddof):
+        np_array1 = np.random.rand(10)
+        np_array2 = np.random.rand(10)
+
+        s1 = Series(np_array1)
```

Comment:
this should be series/methods (split the test in 2)

Reply:
done